corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -454,6 +454,10 @@ def _filter_form_diffs(doc_type, diffs):
     ]
     filtered = _check_deprecation_date(filtered, doc_type)
     filtered = _check_deletion_fields_date(filtered, doc_type)
+    filtered = [
+        diff for diff in filtered
+        if not (diff.diff_type == 'type' and diff.path == ('openrosa_headers', 'HTTP_X_OPENROSA_VERSION'))
```

Comment:
No, it's a type diff related to how the JSON is generated. In one case it's a Decimal and in the other it's a string